sova/llm/client.py

```
@@ -68,34 +70,73 @@ async def invoke(
-        prompt, model=resolved, fallback_model=fallback_model, cwd=cwd, max_budget_usd=max_budget_usd, timeout=timeout
+        prompt,
+        model=resolved,
+        fallback_model=fallback_model,
+        cwd=cwd,
+        max_budget_usd=max_budget_usd,
+        timeout=resolved_timeout,
     )
 
 
-def _resolve_task_type_model(model: str | None, task_type: str | None, *, cwd: Path | str | None = None) -> str | None:
+def _try_load_config(cwd: Path | str | None = None) -> "ProjectConfig | None":
```

Ruff (0.16.1)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)